app/src/main/java/com/codesoom/assignment/JSONParser.java

```diff
@@ -0,0 +1,31 @@
+package com.codesoom.assignment;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+/**
+ * Class that can convert Task Object or List<Task> to JSON style String
```

Comment:
JavaDoc 주석에서 `Task`, `List<Task>`와 같이 타입을 직접 명시하는 것은 커플링이 생겨 바람직하지 않습니다. 예를 들어 `Task` 클래스의 이름을 바꾸게 되면 이 주석도 바꿔야 하기 때문입니다. 이런 주식이 100 개라면 클래스 이름을 바꿀 때 일일이 찾아다니며 바꿔야 하는 일이 생길 수도 있습니다. 주석은 짧고 간결하게, 코드에 의존하지 않으면서도 이해하기 쉽게 적는 것이 바람직합니다.
```suggestion
 * 할 일이나 할 일 목록을 JSON 문자열로 변환하는 기능을 제공합니다.
```